Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -85,6 +85,20 @@
         <item>8</item>
     </integer-array>
 
+    <!-- Log Verbosity selection -->
+    <string-array name="logVerbosityEntries" translatable="false">
```

Comment:
Is there a reason it's missing the "Debug" verbosity and their values don't match the one from `Common/Logging/Log.h`?
```c++
enum LOG_LEVELS
{
  LNOTICE = 1,   // VERY important information that is NOT errors. Like startup and OSReports.
  LERROR = 2,    // Critical errors
  LWARNING = 3,  // Something is suspicious.
  LINFO = 4,     // General information.
  LDEBUG = 5,    // Detailed debugging - might make things slow.
};
```

Reply:
Originally I left out debug logging since that was only available on debug builds. We already warn that logging can slow things down so I might as well add it. And I forgot that LOG_LEVELS start at 1. Changed to match the C++ code.

Reply:
I just remembered to update the default value too.